shell/platform/darwin/ios/framework/Source/FlutterPlatformPluginTest.mm

```diff
@@ -29,6 +29,10 @@ - (void)presentViewController:(UIViewController*)viewControllerToPresent
 
 @implementation FlutterPlatformPluginTest
 - (void)testSearchWebInvoked {
+  id mockApplication = OCMClassMock([UIApplication class]);
+  OCMStub([mockApplication sharedApplication]).andReturn(mockApplication);
+  OCMStub([mockApplication openURL:[OCMArg any]]).andReturn(nil);
```

Comment:
i thikn you also need to verify if openURL actually gets called

Reply:
oh btw, if im not mistaken, i remember we wanted to test the URL escaping as well? (discussed in previous PR)

Reply:
I dont think theres a point in testing URL escaping because Im just calling https://developer.apple.com/documentation/foundation/nsstring/1411946-stringbyaddingpercentencodingwit on the given text